bindings/python/pymongoarrow/lib.pyx

```diff
@@ -31,15 +32,14 @@ from pyarrow.lib import (
 
 from pymongoarrow.errors import InvalidBSON, PyMongoArrowError
 from pymongoarrow.context import PyMongoArrowContext
-from pymongoarrow.types import _BsonArrowTypes, _atypes, ObjectIdType, Decimal128StringType, BinaryType
+from pymongoarrow.types import _BsonArrowTypes, _atypes, ObjectIdType, Decimal128Type as Decimal128Type_, BinaryType
```

Comment:
Can't this just be `ObjectIdType, Decimal128Type, ....`

Reply:
Because there is an arrow [Decimal128Type](https://arrow.apache.org/docs/python/generated/pyarrow.Decimal128Type.html#pyarrow.Decimal128Type) that we'd be overshadowing.

Reply:
Should we consider using another name in that case?

Reply:
Perhaps change them all to `FooExtType`?